src/style.css

```
@@ -0,0 +1,108 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0 8px;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.big__form {
+  height: 70px;
+  margin-top: 20px;
+}
+
+.small__form {
+  height: 42px;
+  margin-top: 20px;
+}
+
+.input__search {
+  height: 100%;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-family: "Avenir" , sans-serif;
+  color: rgb(61 , 78 , 97);
+  font-weight: 300;
+}
+
+.input__search:focus {
+  outline: none;
+  font-weight: 800;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  color: rgb(61 , 78 , 97);
+}
+
+.input__search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input__big {
+  padding-left: 62px;
+  font-size: 16px;
+}
+
+.input__big::placeholder {
+  font-weight: 300;
+  font-size: 16px;
+  color: rgb(61 , 78 , 97);
+}
+
+.input__small::placeholder {
+  font-weight: 300;
+  font-size: 14px;
+  color: rgb(61 , 78 , 97);
+}

```

you can merge these styles to:
```suggestion
.input__search::placeholder {
  font-weight: 300;
  font-size: 16px;
  color: rgb(61 , 78 , 97);
}
```